.github/workflows/ci.yml

```diff
@@ -99,6 +99,14 @@ jobs:
           password: test
           database: test
           rights: --superuser
+      - name: Log DATABASE_URL string
+        shell: bash
+        run: |
+          echo "DATABASE_URL=$DATABASE_URL"
+          echo "And in base64 to bypass Github's obfuscation:"
+          echo "$DATABASE_URL" | base64
```

Comment:
What is it for?

Reply:
Turns out the database_url produced by the test postgres setup is not `postgres://...` but `postgresql://...` -- so my multi-PG code was failing.  I don't really need it anymore.

Reply:
are you ok to keep it?  this is not leaking any secrets -- if you look at the output of that CI step, it outputs all the passwords in clear text too -- its just that github masks anything that looks like a password inside a standard connection string